test/e2e/network/fixture.go

```diff
@@ -114,33 +113,10 @@ func (t *TestFixture) DeleteService(serviceName string) error {
 // Cleanup cleans all ReplicationControllers and Services which this object holds.
 func (t *TestFixture) Cleanup() []error {
 	var errs []error
-	for rcName := range t.rcs {
-		ginkgo.By("stopping RC " + rcName + " in namespace " + t.Namespace)
-		err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
-			// First, resize the RC to 0.
-			old, err := t.Client.CoreV1().ReplicationControllers(t.Namespace).Get(context.TODO(), rcName, metav1.GetOptions{})
-			if err != nil {
-				if apierrors.IsNotFound(err) {
-					return nil
-				}
-				return err
-			}
-			x := int32(0)
-			old.Spec.Replicas = &x
-			if _, err := t.Client.CoreV1().ReplicationControllers(t.Namespace).Update(context.TODO(), old, metav1.UpdateOptions{}); err != nil {
-				if apierrors.IsNotFound(err) {
-					return nil
-				}
-				return err
-			}
-			return nil
```

Comment:
interesting, why this was scaling this down to zero first?

Reply:
I'm not 100% certain, but seeing line 141 it might have been some kind of "scale to 0 first and wait so the deletion is instantaneous afterwards" idea that was never actually implemented. and since it wasn't even implemented I see no benefit otherwise in keeping that logic

Reply:
let's try, I don't fully understand it either, so I tend to agree with you unless we are missing something , better to undraft the PR so we can check the results on the CI

Reply:
I'm still missing some ReplicationControllers in some tests that didn't use `jig.RunRC`, I should start tackling those in this PR, shouldn't I?

Reply:
ok